package_control/package_manager.py

```diff
@@ -1754,7 +1754,8 @@ def print_messages(self, package, package_dir, is_upgrade, old_version, new_vers
 
         def read_message(message_path):
             with open_compat(message_path, 'r') as f:
-                return '\n  %s\n' % read_compat(f).rstrip().replace('\n', '\n  ')
+                lines = read_compat(f).rstrip().split('\n')
+                return '\n' + '\n'.join(['  ' + s if s else '' for s in lines])
```

Comment:
```suggestion
                return '\n' + '\n'.join('  ' + s if s else '' for s in lines)
```

Generator expressions are iterable as well.